public-ui/src/app/views/survey-view/survey-view.component.html

```diff
@@ -141,9 +141,17 @@ <h5 class="box-title">
         <div class="row  answers" [hidden]="!showAnswer[q.conceptId]">
           <div class="col-md-7 col-sm-5 col-xs-12 answer-list">
             <div  class="row answer-text"  [class.answer-active]="q.selectedAnswer && q.selectedAnswer.id === a.id" (click)="showAnswerGraphs(q, a)" *ngFor="let a of q.countAnalysis.results">
-              <div class="col-sm-6 col-xs-12">
+              <div class="col-sm-6 col-xs-12" *ngIf="binnedSurveyQuestions.indexOf(a.stratum2) === -1; else binnedSurveyQuestionDisplay">
                 <label class="answer-text" innerHtml="{{a.stratum4 | highlightSearch : searchText.value }} -- {{a.stratum3}}"> {{a.stratum4}} </label>
               </div>
+              <ng-template #binnedSurveyQuestionDisplay>
+                <div class="col-sm-6 col-xs-12" *ngIf="binnedSurveyQuestions.indexOf(a.stratum2) > -1">
+                  <label class="answer-text" *ngIf="a.stratum4.toLowerCase().trim() === 'did not answer'; else rangeLabel" innerHtml="{{a.stratum4 | highlightSearch : searchText.value }} -- {{a.stratum3}}"> {{a.stratum4}} </label>
```

Comment:
Existing issue, but I noticed during review: https://precisionmedicineinitiative.atlassian.net/browse/DB-129

Reply:
Yeah this was used to highlight text. I was thinking of change that too. Will address in next pr.